### PR TITLE
Structure To-Do-List

### PR DESCRIPTION
**Pull request by Aduda-Boaz**
This PR is for the to-do-list structure, 
All the requirements have been implemented as per the guidelines.

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+.input-task {
+  border-top-color: black;
+  border-top-width: 1px;
+  border-top-style: solid;
+}
+
+.tasks-itm {
+  list-style-type: none;
+  display: flex;
+}
```
Comment (andersonlebon) [inline src/style.css]: One of the requirements states that CSS is created used to style the list using the [captured video](https://www.youtube.com/watch?v=AcUd-_Yjjqg). Please note that the video has more features than the current project, students don't need to style all the features not implemented yet in the video. CSS must be loaded by *Webpack Style/CSS Loader*. here 👇 
image

- [x] Kindly you need to follow the styling in the video demonstration as the requirement says. here is your design 👇 
image

**Review by andersonlebon — CHANGES_REQUESTED** [review]
### Hi @Aduda-Boaz 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there


## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 
- Webpack is used to set up the project ✔️ 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Aduda-Boaz** [comment]
Fix the styling of the application to match the design.

**Review by leolpaz — APPROVED** [review]
Hi @Aduda-Boaz , 

Your project is complete!
Congratulations! 🎉🎉 

Now there is nothing left to do but to merge it! 🏆 





Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**